src/Onboarding/Setup/templates/action-button.html

```diff
@@ -0,0 +1 @@
+<a class="button button-primary" href="{{ href }}" target="{{ target }}">{{ text }}</a>
```

Comment:
I noticed we aren't setting the target value when calling it on the `index.html.php` file, so the link is opened on the same tab/window. 

Is this the expected behavior?

I was wondering if it should be opened in the new tab/window so the user can have both pages open easily or if it should be kept working like now.